packages/sdk/src/grpc/translation.ts

```diff
@@ -550,6 +552,17 @@ function trChainParametersV2(params: v2.ChainParametersV2): v1.ChainParametersV2
 
 export function blockChainParameters(params: v2.ChainParameters): v1.ChainParameters {
     switch (params.parameters.oneofKind) {
+        case 'v3': {
+            const { version, ...common } = trChainParametersV2(params.parameters.v3);
+            const cp: v1.ChainParametersV3 = {
+                ...common,
+                version: 3,
+                validatorScoreParameters: {
+                    maxMissedRounds: unwrap(params.parameters.v3.validatorScoreParameters?.maximumMissedRounds),
+                },
+            };
+            return cp;
```

Comment:
Suggestion: factor into `trChainParametersV3`.

Reply:
Yeah maybe that was too lazy 🙈